source/adapters/hip/usm.cpp

```diff
@@ -148,45 +148,56 @@ ur_result_t USMHostAllocImpl(void **ResultPtr,
   return UR_RESULT_SUCCESS;
 }
 
+static std::optional<hipPointerAttribute_t>
```

Comment:
Nit: I think in general we use anonymous namespace rather than static, although this is sort of matter of style

Reply:
I'll go with that, thanks